### PR TITLE
Feat/new loader

### PR DESCRIPTION
**Pull request by rayyan224**

### CODE REVIEW

File: src/components/Loading/Loading.tsx [hunk 1]
```diff
@@ -1,32 +1,24 @@
...
-                ballColor={hexToRgb(ballColor)}
-                data-testid={size}
-                ringColor={hexToRgb(ringColor)}
-                role={'spinner'}
-                size={size}
-            />
-            {text != '' && <h4>{text}</h4>}
-        </StyledSpinnerParentDiv>
+            <StyledSpinnerDiv spinnerColor={spinnerColor} size={size} />
+            {text != '' && <span>{text}</span>}
```
Comment (BillyG83) [inline src/components/Loading/Loading.tsx:20]: no big deal but you don't need the check as an empty string is falsey
`{text && <span>{text}</span>}`

File: src/components/Loading/types.ts [hunk 2]
```diff
@@ -2,15 +2,12 @@ export interface ILoadingProps {
...
-    // The color of the spinner's ball
-    ballColor?: string;
-
-    // Text to be displayed underneath loader
+    // To load text with spinner
     text?: string;
 
-    // To place text component in column format or row format
-    layout?: 'column' | 'row';
+    // Wherte to place loaded text
```
Comment (locothedev) [inline src/components/Loading/types.ts:11]: TYPOOOOOOOO . Just fix this and merge. Thanksss

**Comment by Y0moo** [comment]
Fixed the typo, merging now

**Comment by rayyan224** [comment]
> Fixed the typo, merging now

@Y0moo completley forgot about this pr . Thanks man 🙏

**Pull request merged**